lib/db/lowlevel.go

```diff
@@ -82,6 +86,14 @@ func NewLowlevel(backend backend.Backend, opts ...Option) *Lowlevel {
 	}
 	db.keyer = newDefaultKeyer(db.folderIdx, db.deviceIdx)
 	db.Add(util.AsService(db.gcRunner, "db.Lowlevel/gcRunner"))
+	if loc := backend.Location(); loc != "" {
+		path := filepath.Join(loc, needsRepairFile)
+		if _, err := os.Lstat(path); err == nil {
+			l.Infoln("Database was marked for repair - this may take a while")
+			db.CheckRepair()
```

Comment:
CheckRepair no longer needs to be exported I think

Reply:
A model test was using it, but luckily that was replaceable by `WithRecheckInterval(time.Millisecond)`.